lldb/source/Target/TargetProperties.td

```diff
@@ -22,6 +22,9 @@ let Definition = "target_experimental" in {
   def SwiftEnableBareSlashRegex: Property<"swift-enable-bare-slash-regex", "Boolean">,
     DefaultFalse,
     Desc<"Passes the -enable-bare-slash-regex compiler flag to the swift compiler.">;
```

Comment:
Can this be removed now, or is this still experimental?

Reply:
last I looked it was still required…